target/linux/ath79/dts/qca9533_yuncore_cpe830.dts

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+       compatible = "yuncore,cpe830", "qca,qca9533";
+       model = "YunCore/KuWfi CPE830(D)";
+
+       chosen {
+               bootargs = "console=ttyS0,115200";
+       };
+
+       keys {
+               compatible = "gpio-keys";
+
+               reset {
+                       label = "reset";
+                       linux,code = <KEY_RESTART>;
+                       gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+               };
+       };
+
+       leds {
+               compatible = "gpio-leds";
+
+               lan {
+                       label = "cpe830:green:lan";
+                       gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+               };
+
+               wan {
+                       label = "cpe830:green:wan";
```

Comment:
Please use the function definition for LEDs:
```suggestion
			color = <LED_COLOR_ID_GREEN>;
			function = LED_FUNCTION_WAN;
```
Do the same for the other LEDs too. Just look for some other device tree files as an example.

Reply:
Where to find the complete lst of " LEF_FUNCTION_xxx"  ?

Reply:
@hauke ?

Reply:
hi @hauke can you help on that ?

Reply:
They are defined in this file: https://elixir.bootlin.com/linux/v6.11/source/include/dt-bindings/leds/common.h I think we backported all recent changes to our kernel 6.6